_layouts/rnative-lesson.html

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+fw: true
+---
+<!-- ^idk what this does but im just putting it -->
```

Comment:
`fw:true` makes the entire layout full-screen, which is what was supposed to happen. We can remove this comment!